src/main/java/com/xpanxion/java/springboot/da1/demo/model/student5/Manager.java

```diff
@@ -0,0 +1,32 @@
+package com.xpanxion.java.springboot.da1.demo.model.student5;
+
+public class Manager {
+    private int manager_id;
+    private String full_name;
```

Comment:
We should probably remove the underscores and use camel case here (managerId).  Underscores are more popular in on the database side, they are shammed on the Java side <insert shame meme here>.

Reply:
Will do. I wasn't sure if they had to match the name in the database exactly.